Lunr/Sniffs/NamingConventions/CamelCapsVariableNameSniff.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Checks the naming of variables and member variables.
+ *
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @copyright 2006-2015 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+ namespace Lunr\Sniffs\NamingConventions;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\AbstractVariableSniff;
+use PHP_CodeSniffer\Util\Common;
+use PHP_CodeSniffer\Util\Tokens;
+
+class CamelCapsVariableNameSniff extends AbstractVariableSniff
+{
+
+
+    /**
+     * Processes this test, when one of its tokens is encountered.
+     *
+     * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
+     * @param int                         $stackPtr  The position of the current token in the
+     *                                               stack passed in $tokens.
+     *
+     * @return void
+     */
+    protected function processVariable(File $phpcsFile, $stackPtr)
```

Comment:
Why not like this?
```suggestion
    protected function processVariable(File $phpcsFile, $stackPtr): void
```

Reply:
I try to modify Sniffs I copied as little as possible beyond the actual behavior I want modified. Makes syncing easier when I don't have to filter out little things like this.